### PR TITLE
DI-375 Modify correlation id character length limit

### PR DESCRIPTION
**Pull request by mik3ola**
## Link to JIRA Ticket
https://nhsd-jira.digital.nhs.uk/browse/DI-375
-

## Description

Dos API currently has a limit 100 on the number of characters for the Correlation Id when invoked. Modifying the Correlation Id character limit in the Utils file on Test suite will facilitate replaying of Changed Events with additional characters on the correlation Id for specific scenarios.

Affected scenarios: F006S008 & F006S009

### Noteworthy Changes

Added new scenario F006S010
Modified Correlation id to accomodate change event replays.

## Type of change

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Testing

- [ ] Integration tests

## Developer Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have run the [code formatting checks](../README.md#code-quality)
- [ ] I have run the [code quality checks](../README.md#code-quality)
- [ ] New code meets [standards](https://nhsd-confluence.digital.nhs.uk/display/DI/DI+Ways+of+Working) agreed by the team
- [ ] Unit test code coverage is at or above 80%
- [ ] New and existing unit tests pass locally with my changes
- [ ] Tests have added that prove my fix is effective or that my feature works (Integration tests)
- [ ] I have made corresponding changes to the documentation
- [ ] I have cleaned down my environment (if created)

## Code Reviewer Checklist

- [ ] I have run the unit tests and they run correctly
- [ ] I can confirm the changes have been tested or approved by a tester
- [ ] I can confirm no remaining infrastructure is left over from this branch

### CODE REVIEW

**Review by rohith1122 — COMMENTED** [review]
All looking good apart from the small issue

File: test/integration/steps/utilities/utils.py [hunk 1]
```diff
@@ -324,7 +324,7 @@ def generate_correlation_id(suffix=None) -> str:
     run_id = getenv("RUN_ID")
     correlation_id = f"{run_id}_{name_no_space}" if suffix is None else f"{run_id}_{suffix}"
     correlation_id = (
-        correlation_id if len(correlation_id) < 100 else correlation_id[:99]
+        correlation_id if len(correlation_id) < 80 else correlation_id[:79]
     )  # DoS API Gateway max reference is 100 characters
```
[the remark this answers is not in the document]
Comment (mik3ola) [reply test/integration/steps/utilities/utils.py:328]: 👍🏼

[the remark this answers is not in the document]
Comment (mik3ola) [reply test/integration/steps/utilities/utils.py:328]: The statement is correct.. We are cutting our test correlation Id to 80 in order to facilitate the 'replay' scenarios which have additional characters added to the corr Id.

**Review by JackPlowman — APPROVED** [review]
LGTM 👍

**Review by mabe13 — APPROVED** [review]

**Pull request merged**